### ISSUE TITLE
Investigate treatment assignment in MORG and if it coincides with the assignment in ASEC

### ISSUE THREAD

**Issue by dgentil1** [dgentil1/Katrina#23]

**Comment by marinabosque** [comment]
- Check that in the **D_Idkatrina_morg.do**, when creating the share of evacuees there is the metcode2 "0". Shouldn't we drop the metcode2 that are "not identified" in the precleaning?

- I think dallas and fayetteville are not in the treatment group....:S

**Comment by dgentil1** [comment]
I've spent most of my after work day in this task today. I've reconstituted the way we were handling metarea codes and changed the pre-cleaning code and its structure extensively. Note that I've had to hand code some part of the match. 
I will keep going tomorrow. 

The good news:

The treatment units are quite the same in both samples!!! :D

But I will translate that finding into code while I try to improve the structure of the overall precleaning and id part. Hopefully soon I will start having some results that we can trust for MORG :D

We can talk whenever you guys want!

Big hug
